Add Reolink reboot button (#28170)
home-assistant/home-assistant.github.io: source/_integrations/reolink.markdown

Not all camera models generate ONVIF push events for all event types, some binary sensors might, therefore, only be polled.
For list of Reolink products that support ONVIF see the [Reolink Support Site](https://support.reolink.com/hc/en-us/articles/900000617826).

> ## General notes for all entities

Reviewer: ```suggestion
## Asterisk (*) next to entities listed in this documentation
```

> Entities listed below that have a * behind their name are disabled by default. To enable them go to settings->devices->reolink->device->+x enties not shown, click the entity you want to enable, settings (gear) -> enable.

Reviewer 2: Is it just me or if I go to the device tab I directly see a list of devices instead of the integrations that provide devices.

Reviewer 2: I think there's probably another way to describe this or something to link to, so I think I'll let someone else who is more familiar with docs review it

Author: Sure, you are right, I was using the Integrations tab instead of the devices tab....
~Will adjust~

Author: ~Does it now make sence to you?~

Reviewer 2: I think so yes, but I still want another docs maintainer to check it.

> Entities listed below that have a * behind their name are disabled by default. To enable them go to settings->devices->select the reolink device->+x enties not shown, click the entity you want to enable, settings (gear) -> enable.

Reviewer: ```suggestion
If an entity listed below has an asterisk (*) next to its name, it means it is disabled by default. To use such an entity, you must [enable the entity](/common-tasks/general/#enabling-entities) first.
```

> ## General notes for all entities
> 
> Entities listed below that have a * behind their name are disabled by default. To enable them go to settings->devices->select the reolink device->+x enties not shown, click the entity you want to enable, settings (gear) -> enable.
> 

Reviewer: ```suggestion
```

- PTZ calibrate
- Guard go to
- Guard set current position

> - Reboot*

Reviewer 2: ```suggestion
- Restart*
```

> - Restart*

Reviewer: ```suggestion
- Restart (Disabled by default. To use it, you must [enable the entity](/common-tasks/general/#enabling-entities) first)
```
